src/status_im/ui/screens/wallet/db.cljs

```diff
@@ -15,3 +19,20 @@
 
 ;; TODO(oskarth): spec for balance as BigNumber
 ;; TODO(oskarth): Spec for prices as as: {:from ETH, :to USD, :price 290.11, :last-day 304.17}
+
+(defn getAmountValidationError [amount web3]
+  (let [amount' (string/replace amount #"," ".")
+        amount-splited (string/split amount' #"[.]")]
+    (cond
+      (or (nil? amount) (= amount "") (= amount "0") (re-matches #"0[,.]0*$" amount))
+      ""
+
+      (or (js/isNaN (js/parseFloat amount'))
+          (try (when (<= (.toWei web3 amount' "ether") 0) true)
+               (catch :default err true)))
+      (i18n/label :t/validation-amount-invalid-number)
+
+      (and (= (count amount-splited) 2) (> (count (last amount-splited)) 18))
+      (i18n/label :t/validation-amount-is-too-small)
```

Comment:
I am not sure I get this one ? When there is more than 18 decimals the amount is considered too small ?

Reply:
right

Reply:
but why is it too small ? maybe too precise ? can't it be 1000.00000000000000...1 or even 0.90000000000000...1 ?

Reply:
yes, you're right, if you're talking about keyword name, again i just take it from bot https://github.com/status-im/status-react/blob/develop/resources/js/bots/wallet/translations.js#L32

Reply:
ah ok I see, can you rename it too-precise then ?

Reply:
yes,sure,thanks